packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/Chart.yaml.patch

```diff
@@ -8,8 +8,8 @@
 +  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
    catalog.cattle.io/release-name: vsphere-cpi
  apiVersion: v1
--appVersion: 1.4.1
-+appVersion: 1.4.2
+-appVersion: 1.26.1
++appVersion: 1.5.1
```

Comment:
Just curious, why we need to override this? The "app" is vSphere CPI chart so it makes sense to track the (latest) version of vSphere CPI plugin included.
This was done in `vsphere-charts` superchart recently - https://github.com/rancher/vsphere-charts/pull/55/files#diff-50ddca5e4037a6988a4f9eb12c30c2c5fa8730fb2cc7d07f1517feee49e01804R11

Reply:
@a-blender , got it! Thank you.